nix/tests.nix

```diff
@@ -214,12 +214,11 @@ let
 in
 fix_names
   {
-    run = test_subdir "run" [ moc ];
+    run-release = test_subdir "run" [ moc ];
     run-debug = snty_subdir "run" [ moc ];
     run-eop-release = enhanced_orthogonal_persistence_subdir "run" [ moc ];
     run-eop-debug = snty_enhanced_orthogonal_persistence_subdir "run" [ moc ];
-    # ic-ref-run = test_subdir "run-drun"   [ moc ic-ref-run ];
-    drun = test_subdir "run-drun" [ moc pkgs.drun ];
+    drun-release = test_subdir "run-drun" [ moc pkgs.drun ];
     drun-debug = snty_subdir "run-drun" [ moc pkgs.drun ];
     drun-compacting-gc = snty_compacting_gc_subdir "run-drun" [ moc pkgs.drun ];
     drun-generational-gc = snty_generational_gc_subdir "run-drun" [ moc pkgs.drun ];
```

Comment:
`snty_` is debug too

Reply:
I know, but they go in this separate gc section now. Before they were in release mode anyhow.